lib/tasks/account.rake

```diff
@@ -27,11 +27,7 @@ namespace :"account" do
 
   desc "Create an account with a preset password"
   task :create_with_password, [ "account", "password" ] => [ "environment" ] do |t,args|
-    unless Conjur::Password.valid?(args[:password])
-      $stderr.puts(::Errors::Conjur::InsufficientPasswordComplexity.new.to_s)
-      exit 1
-    end
-
+    Rake::Task["password:validate_password_strength"].invoke(args[:password])
```

Comment:
i think the task can be named `validate_strength` so we don't have the word `password` duplicated.

Reply:
Done